x-pack/plugins/actions/server/authorization/actions_authorization.ts

```diff
@@ -73,10 +80,21 @@ export class ActionsAuthorization {
           ? operationAlias[operation](authorization)
           : [authorization.actions.savedObject.get(ACTION_SAVED_OBJECT_TYPE, operation)];
 
-        const { hasAllRequested } = await checkPrivileges({
-          kibana: [...privileges, ...additionalPrivileges],
+        privileges.push(...additionalPrivileges);
+
+        const privilegesCheckResponse = await checkPrivileges({
+          kibana: privileges,
         });
-        if (!hasAllRequested) {
+
+        if (!privilegesCheckResponse.hasAllRequested) {
+          logger.info(
+            `Authz for user [${privilegesCheckResponse.username}] failed: ${JSON.stringify(
+              privilegesCheckResponse,
```

Comment:
Does the `privilegesCheckResponse` contain sensitive information that should not be logged?

Reply:
no - it just reports back all of the privileges that you passed in along with ether it failed or passed authz. Here is what the output looks like:

```json5
[2023-11-01T12:41:11.995-04:00][INFO ][plugins.actions.sentinelone] Authz for user [paul] failed: {
  "hasAllRequested": false,
  "username": "paul",
  "privileges": {
    "kibana": [
      {
        "resource": "default",
        "privilege": "saved_object:action/get",
        "authorized": true
      },
      {
        "resource": "default",
        "privilege": "saved_object:action_task_params/create",
        "authorized": true
      },
      {
        "resource": "default",
        "privilege": "api:securitySolution-writeHostIsolationRelease",
        "authorized": true
      },
      {
        "resource": "default",
        "privilege": "fake-one",
        "authorized": false
      }
    ],
    "elasticsearch": {
      "cluster": [],
      "index": {}
    }
  }
}
```

Reply:
Thanks Paul. If I remember correctly the username and the space are considered sensitive. @azasypkin is that the case?  Are we certain that `privilegesCheckResponse` will remain safe to log?

Reply:
> If I remember correctly the username and the space are considered sensitive.

Yes, this is correct. Thanks for checking! It's sensitive information that we reserve exclusively for audit logs. We should also aim to use `INFO` logs sparingly, only when it's truly necessary, to avoid overwhelming our logging output.

Reply:
Ok. I will delete this `log` entry revert the changes that allowed the `logger` to be passed down. Thanks for the feedback